### PR TITLE
Cleaned up code and began support for new WashU data

### PR DESCRIPTION
**Pull request by sgass**

### CODE REVIEW

File: mainWindow.py [hunk 1]
```diff
@@ -25,59 +25,62 @@ def __init__(self, parent=None):
 
         self.sourceModel = QStandardItemModel(self.sourceListView)
         self.sourceListView.setModel(self.sourceModel)
-
+        
         self.t1Series = []
 
+        #Add options to dataTypeComboBox
+        self.dataTypeComboBox.addItems(["Dixon Format","Wash U v1","Wash U v2"])
```
Comment (addisonElliott) [inline mainWindow.py:32]: I would add these strings from Designer. Right-click the combo box in Qt Designer and select Edit Items. Then enter the items

File: dicom2/loadDirectory.py [hunk 2]
```diff
@@ -39,6 +40,7 @@ def loadDirectory(path, patientID=None, studyID=None, seriesID=None):
                 patient = patients[DCMImage.PatientID]
             else:
                 patient = patients.add(DCMImage)
+
```
Comment (addisonElliott) [inline dicom2/loadDirectory.py:43]: We might want to look into code formatting for you. In PyCharm, I code format it a certain way to be consistent.

One blank line between code unless it is between two functions, then it is two blank lines.

Remove the blank line here and above

File: mainWindow.py [hunk 3]
```diff
@@ -25,59 +25,62 @@ def __init__(self, parent=None):
 
         self.sourceModel = QStandardItemModel(self.sourceListView)
         self.sourceListView.setModel(self.sourceModel)
-
+
```
Comment (addisonElliott) [inline mainWindow.py:28]: Might want to remove this tab. I wouldn't have any unnecessary whitespace

File: mainWindow.py [hunk 4]
```diff
@@ -88,10 +91,14 @@ def on_runButton_clicked(self):
         #                         "before converting.")
         #     return
 
-        dataPath = "C:/Users/Clint/PycharmProjects/SIUE-Dixon-Fat-Segmentation-Algorithm/MRI_Data/MF0302-PRE/"
+        #dataPath = "/home/somecallmekenny/SIUE-Dixon-Fat-Segmentation-Algorithm/Data/Subject0001_Final/"
```
Comment (addisonElliott) [inline mainWindow.py:94]: somecallmekenny?

Should I call you kenny?

File: mainWindow.py [hunk 5]
```diff
@@ -108,37 +115,47 @@ def on_runButton_clicked(self):
...
+        dataPath = selectedIndices[0].data()
+        #dataPath = "/home/somecallmekenny/SIUE-Dixon-Fat-Segmentation-Algorithm/Data/Subject0001_Final/"
+
 
         # Load data from path
-        image, config = self.loadFile(dataPath)
-        if image is None:
-            return
+        # Use Correct Load File function based on datatype
+        DataTypeIndex = self.dataTypeComboBox.currentIndex()
```
Comment (addisonElliott) [inline mainWindow.py:142]: I would name the variable dataTypeIndex.

It can be tough to adjust to certain coding styles, but I would stick with camelCase to be consistent.

File: mainWindow.py [hunk 6]
```diff
@@ -108,37 +115,47 @@ def on_runButton_clicked(self):
...
 
         # Load data from path
-        image, config = self.loadFile(dataPath)
-        if image is None:
-            return
+        # Use Correct Load File function based on datatype
+        DataTypeIndex = self.dataTypeComboBox.currentIndex() 
+        if DataTypeIndex == 0:
+            fatImage, waterImage, config = self.loadOldDixonFile(dataPath)
+            self.configureWindow = ConfigureWindow(fatImage, waterImage, config, dataPath, parent=self)
```
Comment (addisonElliott) [inline mainWindow.py:145]: This is good. I would rename ConfigureWindow to ConfigureWindowV1...this leaves room for ConfigureWindowV2..V3.

File: mainWindow.py [hunk 7]
```diff
@@ -108,37 +115,47 @@ def on_runButton_clicked(self):
...
+        elif DataTypeIndex == 1:
+            image, config = self.loadT1File(dataPath)
+        else:
+            QMessageBox.warning(self, "Problem With Data In dataTypeDropBox","Fix index issue.")
```
Comment (addisonElliott) [inline mainWindow.py:149]: Add space between 2nd and 3rd argument

File: mainWindow.py [hunk 8]
```diff
@@ -193,6 +210,72 @@ def loadFile(self, dataPath):
...
+        constants.pathDir = dataPath
+
+        constants.nrrdHeaderDict = {'space': 'right-anterior-superior'}
+        constants.nrrdHeaderDict['space directions'] = (niiFatUpper.header['srow_x'][0:-1],
+                                                        niiFatUpper.header['srow_y'][0:-1],
+                                                        niiFatUpper.header['srow_z'][0:-1])
+
+        constants.nrrdHeaderDict['space origin'] = (niiFatUpper.header['srow_x'][-1],
+                                                    niiFatUpper.header['srow_y'][-1],
+                                                    niiFatUpper.header['srow_z'][-1])
```
Comment (addisonElliott) [inline mainWindow.py:278]: Is this working?

File: mainWindow.py (hunk 1 above)
Comment (sgass) [reply mainWindow.py:32]: Thanks for the tip!

File: dicom2/loadDirectory.py (hunk 2 above)
Comment (sgass) [reply dicom2/loadDirectory.py:43]: I agree, I'd like to stay consistent with what you already have in place. Makes it easier to read the code as a whole. I'll also start using camelCase as mentioned below

File: mainWindow.py (hunk 4 above)
Comment (sgass) [reply mainWindow.py:94]: lol it was a nickname in highschool and have used it as a screen name ever since. You can call me Sam
